packages/slice-machine/server/src/api/services/uploadScreenshotClient.ts

```diff
@@ -57,7 +50,7 @@ export async function upload(
     console.error(msg);
     console.error(`Full error: ${JSON.stringify(aclResponse)}`);
     // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-argument
-    return onError(aclResponse, msg);
+    return { err: onError(aclResponse, msg) };
```

Comment:
what does onError do?

Reply:
onError returns the usual object that we expect { err, reason, status }